redash/models/parameterized_query.py

```diff
@@ -36,6 +36,13 @@ def dropdown_values(query_id):
     return map(pluck, data["rows"])
 
 
+def resolve_parameter_values(parameters):
```

Comment:
Again, I feel that this function name is too generic for what it does. There's no "resolving" here, but rather flattening of multiple values.